src/components/Votes.jsx

```diff
@@ -0,0 +1,52 @@
+import { patchVotes } from '../utils/api';
+import { useState } from 'react';
+import voteUpImage from '../images/voteUpArrow.png';
+import voteDownImage from '../images/voteDownArrow.png';
+
+const Votes = ({ votes, article_id }) => {
+	const [voteIncrement, setVoteIncrement] = useState(0);
+
+	const incrementVotesUp = () => {
+		setVoteIncrement((currVoteIncrement) => {
+			return currVoteIncrement + 1;
+		});
+		patchVotes(article_id).catch(() => {
+			setVoteIncrement((currVoteIncrement) => {
+				return currVoteIncrement - 1;
+			});
+		});
+	};
+	const incrementVotesDown = () => {
+		setVoteIncrement((currVoteIncrement) => {
+			return currVoteIncrement - 1;
+		});
+		patchVotes(article_id).catch(() => {
+			setVoteIncrement((currVoteIncrement) => {
+				return currVoteIncrement + 1;
+			});
+		});
+	};
```

Comment:
Your `incrementVotesUp` and `incrementVotesDown` look very similar. Try to refactor this and use only one function to increment or decrement votes. Hint: Use arguments!